GitUI/CommandsDialogs/SettingsDialog/Pages/GeneralSettingsPage.cs

```diff
@@ -47,6 +47,9 @@ public GeneralSettingsPage()
             cboDefaultPullAction.ValueMember = "Value";
             cboDefaultPullAction.DataSource = pullActions;
             cboDefaultPullAction.SelectedIndex = 0;
+
+            cboEditor.Items.Add(string.Empty);
+            cboEditor.Items.AddRange(EditorHelper.GetEditors());
```

Comment:
I don't think this is the right settings page for the setting. Besides, the wrong context, the page is already at the size limit for small factor screens.

Reply:
I looked at all the other setting pages, and none of them seem to remotely fit. Except maybe "Advanced", but it's not a particularly advanced feature.

Reply:
Moved the setting to the Diff viewer page, which has a lot more room. Even though commit dialog and file list are also influenced, the prior doesn't directly link to its settings, and the latter doesn't have a settings page at all.